freqtrade/resolvers/exchange_resolver.py

```diff
@@ -32,23 +32,17 @@ def _load_exchange(
         :param extra_dir: additional directory to search for the given exchange
         :return: Exchange instance or None
         """
-        current_path = Path(__file__).parent.parent.joinpath('exchange').resolve()
-
-        abs_paths = [
-            current_path.parent.parent.joinpath('user_data/exchange'),
-            current_path,
-        ]
-
-        for _path in abs_paths:
-            try:
-                exchange = self._search_object(directory=_path, object_type=Exchange,
-                                               object_name=exchange_name,
-                                               kwargs=kwargs)
-                if exchange:
-                    logger.info('Using resolved exchange %s from \'%s\'', exchange_name, _path)
-                    return exchange
-            except FileNotFoundError:
-                logger.warning('Path "%s" does not exist', _path.relative_to(Path.cwd()))
+        abs_path = Path(__file__).parent.parent.joinpath('exchange').resolve()
+
+        try:
+            exchange = self._search_object(directory=abs_path, object_type=Exchange,
+                                           object_name=exchange_name,
+                                           kwargs=kwargs)
+            if exchange:
+                logger.info('Using resolved exchange %s from \'%s\'', exchange_name, abs_path)
```

Comment:
Please use double-quotes instead of escaping ...

I suspect it's wrong in the other resolvers as well :( sorry for that ...

Reply:
No worries, I'll update all of them